solidity/contracts/bridge/library/Fraud.sol

```diff
@@ -0,0 +1,396 @@
+// SPDX-License-Identifier: MIT
+
+// ██████████████     ▐████▌     ██████████████
+// ██████████████     ▐████▌     ██████████████
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+// ██████████████     ▐████▌     ██████████████
+// ██████████████     ▐████▌     ██████████████
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+
+pragma solidity ^0.8.9;
+
+import {BytesLib} from "@keep-network/bitcoin-spv-sol/contracts/BytesLib.sol";
+import {BTCUtils} from "@keep-network/bitcoin-spv-sol/contracts/BTCUtils.sol";
+import {CheckBitcoinSigs} from "@keep-network/bitcoin-spv-sol/contracts/CheckBitcoinSigs.sol";
+import "../../GovernanceUtils.sol";
+import "./EcdsaLib.sol";
+import "../Bridge.sol";
+
+library Fraud {
+    using BytesLib for bytes;
+    using BTCUtils for bytes;
+    using BTCUtils for uint32;
+    using EcdsaLib for bytes;
+
+    struct FraudChallenge {
+        // The address of the party challenging the wallet.
+        address challenger;
+        // The amount of ETH the challenger deposited.
+        uint256 depositAmount;
+        // The timestamp the challenge was submitted at.
+        uint32 reportedAt;
+        // The flag indicating whether the challenge has been closed.
+        bool closed;
+    }
+
+    struct Data {
+        ///  The amount of stake slashed from each member of a wallet for a fraud.
+        uint256 slashingAmount; //TODO: Initialize
+        /// The percentage of the notifier reward from the staking contract
+        /// the notifier of a fraud receives.
+        uint256 notifierRewardMultiplier; //TODO: Initialize
+        /// The amount of time the wallet has to defend against a fraud challenge.
+        uint256 challengeDefendTimeout; //TODO: Initialize
```

Comment:
Let's be consistent and name it `challengeDefeatTimeout` as the entire nomenclature is built around defeating not defending the challenge.

Reply:
I made naming consistent and used "defeat" instead of "defend" everywhere in bb8535061e0a713f9c5e478694d50df1d3fe3138.